tools/actions/composites/update-snapshots-desktop/action.yml

```diff
@@ -18,13 +18,16 @@ runs:
       shell: bash
     - name: status
       id: status
-      run: echo "status=$(git status --porcelain | wc -l)" >> $GITHUB_OUTPUT
+      run: echo "status=$(git status --porcelain)" >> $GITHUB_OUTPUT
```

Comment:
please don't remove the `wc -l` command as we are using it in the `if:` statement of `Commit snapshots`